### DISCUSSION TITLE
Where Can i Find App_Start Folder?

### DISCUSSION THREAD

**Discussion by ahmadfauzanarif** [serenity-is/Serenity#6230] [Help]
Hello, I want to ask. Where can I find the App_Start folder? I only found the App_Data folder.
Thanks for the question. hope it's answered

**Comment by wezmag** [comment]
There is no App_Start folder in the new .NET 5 version.
However, you can find the Startup.cs file in the Initialization folder.
![image](https://user-images.githubusercontent.com/340794/150303033-3fbf35e0-8c3c-4b22-a74a-0209ea2102b4.png)

**Reply by ahmadfauzanarif** [reply]
Thank a lot. You Help me

**Reply by ahmadfauzanarif** [reply]
But I haven't found what I'm looking for.
I want to search this.

https://github.com/serenity-is/Serenity/wiki/Database-MySQL-Connection-Timeout-on-Northwind-Database-creation-during-site-initialization

**Reply by wezmag** [reply]
In `DataMigrations.cs`, find below code.
```cs
var runner = scope.ServiceProvider.GetRequiredService<IMigrationRunner>();
runner.MigrateUp();
```
and change to 
```cs
var runner = scope.ServiceProvider.GetRequiredService<IMigrationRunner>();
runner.RunnerContext.Timeout = 100; //change to whatever you need.
runner.MigrateUp();
```

I am not sure if this is the correct approach. You can give a try.

**Reply by wezmag** [reply]
Sorry, I just tried. It does not work. :(

**Reply by wezmag** [reply]
Try this one.
In `DataMigrations.cs`, find below code and change it.
``` cs
.Configure<ProcessorOptions>(options =>
{
    options.Timeout = TimeSpan.FromSeconds(90);
})
```

**Reply by ahmadfauzanarif** [reply]
Thank you. I will try it first
